src/controllers/GameController.ts

```diff
@@ -1,17 +1,21 @@
-import { Colors } from '../enums';
+import { Colors, PieceNames } from '../enums';
 import { Square } from '../models/Square';
 import { GameEngine } from '../services/game-logic/GameEngine';
 import { Piece } from '../services/game-logic/pieces/Piece';
 import { BoardView } from '../views/BoardView';
+import { ModalView } from '../views/ModalView';
+import { PawnPromotionView } from '../views/PawnPromotionView';
 
 class GameController {
     boardView: BoardView;
     gameEngine: GameEngine;
     activeSquare: Square | null;
     currentPlayer: Colors;
+    modal: ModalView | null;
```

Comment:
to jest chyba jakiś punkt zaczepny z modalem Lilii - to zostanie jakoś połączone?

Reply:
Konrad to ja zmienię u siebie nazwy plików/klas, zmiennych, żeby ten modal nam się tak nie powtarzał

Reply:
i bardzo ładnie ten modal wygląda, mam nadzieję, że też dobrze wyjdzie z layoutem, bo ta półprzezroczysta plansza może jednak nie łączyć się dobrze z takim modalem, ale to zobaczymy w praniu

Reply:
Oks, to narazie zostawiam :)